### PR TITLE
Add cloud resources for Datadog integration

### PR DESCRIPTION
**Pull request by lordsarcastic**
# Description
Added a guide on how to ingest cloud resources using Datadog integration

## Added docs pages
- Ingest cloud resources using Datadog (`docs/guides/all/ingest-cloud-resources-using-datadog.md`)

### CODE REVIEW

File: docs/guides/all/ingest-cloud-resources-using-datadog.md [hunk 1]
```diff
@@ -0,0 +1,147 @@
...
+- You have installed integrations for [GCP](https://docs.datadoghq.com/integrations/google_cloud_platform/?tab=project), [AWS](https://docs.datadoghq.com/integrations/#cat-aws) and [Azure](https://docs.datadoghq.com/integrations/azure/) on your Datadog environment
+
+
+## Ingesting cloud resources into Port
+We will be making use of the `hosts` kind in Port's Datadog integration which provides information on hosts existing on the cloud providers we have configured. So far, we are only interested in hosts with `gcp`, `aws`, and `azure` in the `sources` property.
+
+1. Having installed the Datadog integration, create the `datadogCloudResource` blueprint in your Port environment using the blueprint below:
+
+<details>
+<summary><b>Datadog Cloud Resource Blueprint (Click to expand</b></summary>
```
Comment (PeyGis) [inline docs/guides/all/ingest-cloud-resources-using-datadog.md:27]:
```suggestion
<summary><b>Datadog Cloud Resource Blueprint (Click to expand)</b></summary>
```

**Review by PeyGis — CHANGES_REQUESTED** [review]
Looks great. Please change the image. Try to show the important properties such as platform, cpu core, source etc

**Review by PeyGis — APPROVED** [review]
LGTM

**Review by PeyGis — CHANGES_REQUESTED** [review]
Let's make the guide appear on the all guides page and make it searchable by tags or title

To make it appear, you need to modify this file
src/components/guides-section/consts.js
and include an entry such as
    {
        title: "Sync Service Entities to Incident IO",
        description: "Create a GitHub action that syncs service entities to Incident IO catalog",
        tags: ["Incident management","GitHub"],
        logos: ["GitHub"],
        link: "/guides/all/sync-service-entities-to-incident-io",
    }

**Review by PeyGis — APPROVED** [review]
nice work

File: docs/guides/all/ingest-cloud-resources-using-datadog.md [hunk 2]
```diff
@@ -0,0 +1,147 @@
...
+
+## Prerequisites
+This guide assumes the following:
+- You have a Port account and that you have finished the [onboarding process](/quickstart).
+- You have [installed and setup Port's Datadog integration](/build-your-software-catalog/sync-data-to-catalog/apm-alerting/datadog.md)
+- You have installed integrations for [GCP](https://docs.datadoghq.com/integrations/google_cloud_platform/?tab=project), [AWS](https://docs.datadoghq.com/integrations/#cat-aws) and [Azure](https://docs.datadoghq.com/integrations/azure/) on your Datadog environment
+
+
+## Ingesting cloud resources into Port
+We will be making use of the `hosts` kind in Port's Datadog integration which provides information on hosts existing on the cloud providers we have configured. So far, we are only interested in hosts with `gcp`, `aws`, and `azure` in the `sources` property.
```
Comment (hadar-co) [inline docs/guides/all/ingest-cloud-resources-using-datadog.md:22]:
```suggestion
We will be making use of the `hosts` kind in Port's Datadog integration which provides information on hosts existing on the cloud providers we have configured. For this example, we are only interested in hosts with `gcp`, `aws`, and `azure` in the `sources` property.
```

File: docs/guides/all/ingest-cloud-resources-using-datadog.md [hunk 3]
```diff
@@ -0,0 +1,147 @@
...
+</details>
+
+:::info Mapping explanation
+Cloud resources from Datadog are hosts from the `host` kind of the Datadog integration which are from GCP, AWS or Azure. To ingest them, we have to specify a selector which allows only such hosts with the sources we specify.
+
+By doing this, we are able to ingest all cloud resources from Datadog
+
+:::
+
+4. Next, click on resync and watch your cloud resources from Datadog being ingested:
```
Comment (hadar-co) [inline docs/guides/all/ingest-cloud-resources-using-datadog.md:145]:
```suggestion
4. Finally, click `resync`. Your cloud resources from Datadog should now be ingested:
```

**Pull request merged**